### ISSUE TITLE
Address Card balance display broken

### ISSUE THREAD

**Issue by Stefie** [polkadot-js/apps#1278]
With a Substrate version from yesterday (https://github.com/paritytech/substrate/commit/a7dd2559a7658f11eed101efdcfc26dd8b57a9b5) the address card display seems to be broken.

The screenshot was taken on from https://polkadot.js.org/apps/#/accounts with a freshly installed node (pruned db, cleaned cargo cache and rebuilt WASM).

<img width="1517" alt="Screenshot 2019-06-05 at 11 43 11" src="https://user-images.githubusercontent.com/125398/58946940-3a4d7400-8787-11e9-80a2-db3037311cb9.png">

**Comment by Stefie** [comment]
Maybe related to https://github.com/paritytech/substrate/commit/46121f7c073f1b6ee34718bcaacf09e0f8fb395a?

**Comment by jacogr** [comment]
Yes, this is not an apps issue - API is to be updated to have the new digest types for substrate master.
